src/components/gtt-client/init/events.ts

```diff
@@ -82,17 +82,17 @@ function handleIssueSelection(this: any): void {
 
 
 /**
- * Handles the click event on the edit icon to update the map size when the editable form is made visible.
+ * Handles the click event on the edit/comment icon to update the map size when the editable form is made visible.
  */
 function handleEditIcon(this: any): void {
-  document.querySelectorAll('div.contextual a.icon-edit').forEach((element: HTMLAnchorElement) => {
+  document.querySelectorAll('div.contextual a.icon-edit, div.contextual a.icon-comment').forEach((element: HTMLAnchorElement) => {
     element.addEventListener('click', () => {
       setTimeout(() => {
         this.maps.forEach((m: any) => {
           m.updateSize();
         });
         zoomToExtent.call(this);
-      }, 200);
+      }, 500);
```

Comment:
@dkastl Thanks for the fix!

In my local environment, keeping this `delay` argument's value to original (`200`) was better. (Without showing sea probably around lon,lat = (0, 0)).
Could you check this part a bit ?

| 500ms | 200ms |
|--------|--------|
| ![redmine-gtt-issue320-setTimeout-delay-500](https://github.com/gtt-project/redmine_gtt/assets/629923/0c914ea0-3f02-44dd-ad34-4248ccb548ed) | ![redmine-gtt-issue320-setTimeout-delay-200](https://github.com/gtt-project/redmine_gtt/assets/629923/b0ef202f-c847-4374-8486-610983b4acdc) |

Reply:
Well, I guess that it depends on the device performance eventually, and you have a relatively fast device. We probably need to check on various devices, but in my case 200ms was not enough for the "quote" link case, not on point features either.

Reply:
@dkastl (CC: @smellman)
Okay, I reproduced the issue when changing 200ms to 0ms on my environment.
Personally, I think that adjusting timeout delay value is not good idea, and using MutationObserver will be better for such case like issues list view collapsible map.
https://github.com/gtt-project/redmine_gtt/blob/main/src/components/gtt-client/init/events.ts#L28-L47

Reply:
Well, if someone knows, how to implement this with MutationObserver, I agree this is probably more reliable solution. Otherwise, this PR does not really introduce the timeout but just increases the value.

Reply:
@dkastl Okay.
I am okay to implement MutationObserver, but it will take a certain time (maybe next week or later), so I approve this.

Reply:
thanks, I created an issue: https://github.com/gtt-project/redmine_gtt/issues/323
... and will merge this one.